src/views/components/onPageLoad/template.njk

```diff
@@ -0,0 +1,17 @@
+<script {% if params.nonce%} nonce='{{ params.nonce }}'{%  endif %}>
```

Comment:
As previously discussed, as this component is not specific to KBVs, in a future PR this should be extracted from this repo and pulled into the shared cri-common-express repository instead.

Reply:
https://govukverify.atlassian.net/browse/DFC-491 - Tech debt ticket for this change